src/explore.js

```diff
@@ -24,8 +24,10 @@ export const exploreSetup = () => {
 };
 
 export const exploreResponse = (type, message) => {
-  chrome.tabs.query({ active: true, url: 'https://*/explore*' }, (tabs) => {
-    chrome.tabs.sendMessage(tabs[0].id, { type, message });
+  chrome.tabs.query({ active: true, url: exploreHost }, (tabs) => {
+    if (tabs.length > 0) {
+      chrome.tabs.sendMessage(tabs[0].id, { type, message });
+    }
```

Comment:
If tabs.length = 0, we should probably some sort of error right? not totally sure how though since this is how we communicate with front end ... so if theres an error communicating with the front end how do we show an error on the front end lol - or maybe thats not necessary?